packages/docs/src/content/charts/01.bar/06.toast.mdx

```diff
@@ -547,6 +409,234 @@ files:
         step,
       };
     }
+
+  XAxis.js: |
```

Comment:
Why did you separate `XAxis.js` and `YAxis.js`? Was it because the layout didn't align properly during rendering?